rebar.config

```diff
@@ -48,4 +48,5 @@
 
 {cover_enabled, true}.
 {cover_print_enabled, true}.
+{plugins, [rebar3_hex]}.
```

Comment:
Don't do this, it'll cause any project that depends on pooler to also fetch `rebar3_hex`. It is best to just put it in `~/.config/rebar3/rebar.config` but if you don't want it in your global config for some reason you can also put it as `{project_plugins, [rebar3_hex]}.`. Project plugins do not get fetched when the application is being used as a dependency. Could also put it in a profile like, `{profiles, [{pkg, [{plugins, [rebar3_hex]}]]]}.`